src/blockchain_processor.py

```diff
@@ -489,15 +480,13 @@ def do_unsubscribe(self, method, params, session):
                 if session in l:
                     print_log("error rc!!")
                     self.shared.stop()
-                if l == []:
-                    self.watched_addresses.pop(addr)
```

Comment:
n/m, didn't realize what the session removal logic was doing above it. Reverting this change.